bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -148,15 +173,15 @@
         <p>{{_('Get all the most popular games and social networking apps right here.')}}</p>
         <ul class="apps-marketplace">
           <li>
-            <img src="{{ static('img/firefox/os-new/icons/facebook.png') }}" width="142" height="142" alt="Facebook" />
+            <div class="icon facebook"></div>
```

Comment:
Same as above, these could possibly be done with pseudo-elements instead of empty divs